src/test/fakeExtensionContext.ts

```diff
@@ -54,14 +56,32 @@ export class FakeExtensionContext implements ExtensionContext {
     public static async getNew(): Promise<FakeExtensionContext> {
         const ctx = new FakeExtensionContext()
         try {
-            ExtensionDisposableFiles.getInstance().dispose()
+            TestExtensionDisposableFiles.clearInstance()
         } catch {
-            await ExtensionDisposableFiles.initialize(ctx)
+            // Ignore.
         }
+        await TestExtensionDisposableFiles.initialize(ctx)
         return ctx
     }
 }
 
+export class TestExtensionDisposableFiles extends ExtensionDisposableFiles {
```

Comment:
What is the value in extending ExtensionDisposableFiles? This class seems like it could be a freestanding test utility class.

Reply:
one reason is that `INSTANCE` is `protected`

Reply:
Ok, I overlooked that